SeaBattle.API/Controllers/EngineController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using EngineNamespace;
+using BoardNamespace;
+using SeaBattle.Engine.Interfaces;
+using SeaBattle.Values;
+using SeaBattle.API.Infrastructure;
+using Microsoft.AspNetCore.Cors;
+using SeaBattle.API.Services;
+using SeaBattle.API.Domains.Engine.Models;
+using SeaBattle.API.Controllers;
+
+namespace WebApp.Controllers
+{
+    [ApiController]
+    [Route("api/seabattle")]
+    [EnableCors("AllowSpecificOrigin")]
+    public class GameController : ControllerBase
+    {
+        private readonly EngineService _engineService;
+        private readonly ILogger<GameController> _logger;
+
+        public GameController(ILogger<GameController> logger, EngineService engine)
```

Comment:
Nitpick: Logger and other technical dependencies are usually located at the end of the constructor, service with business logic should be first.